src/core/PWSfile.h

```diff
@@ -29,7 +29,7 @@
 // HASH_ITERATIONS is used by the key stretching algorithm.
 // MIN_HASH_ITERATIONS is a lower limit - anything lower than this
 // is considered inherently insecure.
-#define MIN_HASH_ITERATIONS 2048
+#define MIN_HASH_ITERATIONS 16384
 // MAX_USABLE_HASH_ITERS is a guesstimate on what's acceptable to a user
 // with a reasonably powerful CPU. Real limit's 2^32-1.
 #define MAX_USABLE_HASH_ITERS (1 << 22)
```

Comment:
This one could go up too, 22 is easy-peasy for my iPhone.  I actually also tried 2^32-1 on the phone, which is not really acceptable (takes between 30-60 seconds), but still it gives an indication that hardware is coming close to the maximum limit.